nix/grpc.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchFromGitHub, cmake, zlib, c-ares, pkgconfig, openssl, protobuf, gflags }:
 
 stdenv.mkDerivation rec {
-  version = "1.22.1";
+  version = "1.35.0";
```

Comment:
This change is a bit more speculative, I've tested outside of nix with this version.
But I don't have a working local nix on OSX to test it.

Reply:
This is sorted with the updates to nix.